cypress/support/commands.js

```diff
@@ -24,6 +24,29 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 // one of the fec dependencies talks to window.insights.chrome
+import React, { Suspense } from 'react';
+import { Provider } from 'react-redux';
+import { init } from '../../src/Store';
+import messages from '../../locales/data.json';
+import IntlProvider from '@redhat-cloud-services/frontend-components-translations/Provider';
+import { mount } from '@cypress/react';
+
+Cypress.Commands.add('mountWithContext', (Component, FallBackComponent) => {
```

Comment:
Can we also support passing props for Component? Like here for example https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/cypress/support/commands.js#L35